textpipe/doc.py

```diff
@@ -274,3 +274,24 @@ def _set_default_nlp(self, lang):
             self._spacy_nlps[lang] = {}
         model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
         self._spacy_nlps[lang][None] = model
+
+    @functools.lru_cache()
+    def sentiment(self, word_map={}):
```

Comment:
yeah, you shouldn't do that. Set the default to None and then below say word_map = word_map or {}